src/main/resources/db/migration/v0.28.0.0__add_missing_stuff.sql

```diff
@@ -0,0 +1,45 @@
+CREATE SEQUENCE public.curationreporthistory_seq  START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.dataprovider_seq           START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.experimentalcondition_seq  START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.genetogeneorthology_seq    START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.note_seq                   START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.organization_seq           START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.person_seq                 START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.personsetting_seq          START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.reagent_seq                START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.resourcedescriptor_seq     START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.resourcedescriptorpage_seq START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.revinfo_seq                START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.slotannotation_seq         START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.synonym_seq                START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.vocabulary_seq             START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.vocabularyterm_seq         START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+CREATE SEQUENCE public.vocabularytermset_seq      START WITH 200000000 INCREMENT BY 50 NO MINVALUE NO MAXVALUE CACHE 1;
+
+DROP SEQUENCE hibernate_sequence;
+
+ALTER TABLE ONLY public.annotation                               ADD CONSTRAINT uk_5yecel87ntgvx62kmhgmlxw1y UNIQUE (modinternalid);
+ALTER TABLE ONLY public.vocabulary                               ADD CONSTRAINT uk_avxkpwi6f4x6fhb7pvsi86pog UNIQUE (vocabularylabel);
+ALTER TABLE ONLY public.reagent                                  ADD CONSTRAINT uk_eakmwvrtkqtnv0j6p5sacjith UNIQUE (modentityid);
+ALTER TABLE ONLY public.reagent                                  ADD CONSTRAINT uk_f9mv5mtgjw7yhjj8c4985i649 UNIQUE (modinternalid);
+ALTER TABLE ONLY public.vocabularytermset                        ADD CONSTRAINT uk_kovuuc5lh1hwyccs4iesimxst UNIQUE (vocabularylabel);
+ALTER TABLE ONLY public.annotation                               ADD CONSTRAINT uk_q3qsfxfry01magx331btt1dg3 UNIQUE (modentityid);
```

Comment:
This constraint already exists as annotation_modentityid_uk

Reply:
added the uk CONSTRAINT to the class